src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/DeleteCompleteFilesTask.java

```diff
@@ -120,29 +114,7 @@ private boolean tryProcessCompleteEnvelope(CloudBlobContainer container, Envelop
 
         } catch (Exception ex) {
             log.error("Failed to process file. {}", loggingContext, ex);
-            registerDeleteFailureEvent(envelope, ex.getMessage());
```

Comment:
We could have alternatively register event only when it is not a storage exception for acquiring lease